manifests/sar_metric.pp

```diff
@@ -0,0 +1,60 @@
+define puppet_metrics_collector::sar_metric (
+  String                    $output_dir,
+  String                    $scripts_dir,
+  Enum['absent', 'present'] $metric_ensure = 'present',
+  String                    $metrics_type = $title,
+  String                    $cron_minute = '*/5',
+  Integer                   $retention_days = 90,
+  Integer                   $polling_frequency_seconds = 1,
+  Integer                   $collection_frequency = 5, #minutes
```

Comment:
This should have a boolean parameter to manage the `sar` package, which defaults to true. Then a gated package resource on the `sar` package on with that parameter. That way we can ensure that the required packages are installed with an option to disable the management of them if they are already in the catalog.

Reply:
I added this at the system level without thinking about where your comment was by accident.  Then I realized you comment was on sar_metric...   I can move it to here, but how would the user access it, since they normally pass parms to the system class...

Reply:
@jarretlavallee I checked with John and he thought this approach was okay because system.pp is the entry point for the whole of the sar stuff.  But I am open to other ideas.  I could in theory take the parameter at the system level and pass it to system_cpu/system_memory and have then pass it to sar_metric for instance...

Reply:
system.pp seems like the best place for it.